src/platform/win.rs

```diff
@@ -92,6 +95,43 @@ impl WV for InnerWebView {
                     Ok(())
                 })?;
 
+                let mut custom_protocol_name = None;
+                if let Some(protocol) = custom_protocol {
+                    // WebView2 doesn't support non-standard protocols yet, so we have to use this workaround
+                    // See https://github.com/MicrosoftEdge/WebView2Feedback/issues/73
+                    custom_protocol_name = Some(protocol.0.clone());
+                    w.add_web_resource_requested_filter(
+                        &format!("file://custom-protocol-{}*", protocol.0),
+                        webview2::WebResourceContext::All,
+                    )?;
+                    w.add_web_resource_requested(move |_, args| {
+                        let uri = args.get_request().unwrap().get_uri().unwrap();
+                        // Remove leading custom protocol indicator
+                        let path = &uri[(23 + protocol.0.len())..];
+                        match protocol.1(path) {
```

Comment:
I should destructure the `if let` so there isn't any `.n` stuff.

Reply:
I'm outside without Windows. I'll make sure to do it back home.

Reply:
I'm outside without Windows. I'll make sure to do it back home.